.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Timecop
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: PHP ${{ matrix.php-version }} (ZTS ${{ matrix.php-zts }})
+    runs-on: ubuntu-20.04
```

Comment:
Do we not always want `ubuntu-latest` instead?

Reply:
That's a moving target.

Reply:
True but it does usually stay LTS and will avoid us having to keep updating it?

Reply:
I think one manual update every 2 to 4 years is acceptable. Leaving it tracking latest could cause the actions to break if the definition of latest changes before the actions are updated to support the new version.